standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/MetaStoreSchemaInfo.java

```diff
@@ -38,10 +38,11 @@
 
 
 public class MetaStoreSchemaInfo implements IMetaStoreSchemaInfo {
-  protected static final String UPGRADE_FILE_PREFIX = "upgrade-";
```

Comment:
I believe this should either be protected or we should provide an method in the IMetaStoreSchemaInfo so that custom implementations of IMetaStoreSchemaInfo can use their own upgrade prefixes.

Reply:
Makes sense.  Should we apply the same logic to the other prefixes as well?

Reply:
Yes, it makes sense to a expose others too.